pkg/standalone/run.go

```diff
@@ -132,6 +133,10 @@ func (config *RunConfig) validate() error {
 		return err
 	}
 
+	if config.ResourcesPath != "" {
+		config.ComponentsPath = ""
```

Comment:
@hueifeng is this intentional?
or do we intend to set `components-path` when `resource-path` is set ?

Reply:
> @hueifeng is this intentional? or do we intend to set `components-path` when `resource-path` is set ?

Yes, this is intentional, what I think is to replace it slowly to `resource-path`, If we copy the code of `component-path` again, it will be a bit repetitive, after all, we only need PR to change the name.